service/src/main/java/tds/assessment/repositories/impl/AccommodationsQueryRepositoryImpl.java

```diff
@@ -282,7 +282,7 @@ public AccommodationsQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTe
                 "  and TType.Context = '*' \n" +
                 "  and TT.ContextType = 'TEST' \n" +
                 "  and TT.Context = '*' \n" +
-                "  and (TType.TestMode = 'ALL' AND TT.TestMode = 'ALL') \n" +
+                "  and (TType.TestMode IN ('ALL', 'online') AND TT.TestMode = 'ALL') \n" +
```

Comment:
Nope, for some reason in the seed data that inserts the TTS IntraTest Control record, the testmode is set to 'online' only in testtooltype. testtool is still 'ALL'